.github/workflows/plugin-release.yml

```diff
@@ -0,0 +1,76 @@
+# Create release on change to plugin.info.txt version line
+# https://github.com/dokuwiki/dokuwiki/issues/3951
+#
+# Requires DOKUWIKI_USER and DOKUWIKI_PASS secrets be set in GitHub Actions
+
+name: Release Plugin
+
+on:
+  workflow_call:
+    secrets:
+      DOKUWIKI_USER:
+        required: true
+      DOKUWIKI_PASS:
+        required: true
+
+env:
+  INFOTXT: plugin.info.txt
+  PHP_VERSION: "8.2"
+
+jobs:
+  release:
+    name: Create release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup name and version
+        id: version
+        run: |
+          VERSION=$(awk '$1 == "date" {print $2}' $INFOTXT)
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+          BASE=$(awk '$1 == "base" {print $2}' $INFOTXT)
+          echo "BASE=$BASE" >> $GITHUB_ENV
+
+      - name: Create release
+        id: create_release
+        uses: actions/create-release@v1
```

Comment:
This action is using deprecated Github API (`set-ouput` https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) and the plugin has been archived signalling that it will not be fixed. The [README](https://github.com/actions/create-release/blob/main/README.md) lists some alternatives

Reply:
@mprins if there's updated version of this, please share it with the rest of us.

so far I'll continue with existing solution for this project:
- https://github.com/saggi-dw/dokuwiki-plugin-htmlok/issues/6